platform/linux-generic/odp_crypto_openssl.c

```diff
@@ -509,8 +509,9 @@ auth_xcbcmac_check(odp_packet_t pkt,
 	odp_packet_copy_to_mem(pkt, param->hash_result_offset,
 			       bytes, hash_in);
 
-	_odp_packet_set_data(pkt, param->hash_result_offset,
-			     0, bytes);
+	if (odp_unlikely(session->p.hash_result_in_auth_range))
+		_odp_packet_set_data(pkt, param->hash_result_offset,
+				     0, bytes);
```

Comment:
These _odp_packet_set_data() calls in this patch should fit into a single line (<100 chars)

Reply:
fixed in v4